docs/docs/unreachable.md

```diff
@@ -0,0 +1,90 @@
+
+---
+title: Unreachable Strategy
+---
+
+# Unreachable Strategy
+
+In order for Marathon to provide partition aware unreachable strategy support there are 2 high level events that must occur; 1) Mesos needs to communicate a task is unreachable and 2) Marathon must respond to that event if unresolved within a specified amount of time. Each of these events have configuration options and DCOS system defaults which are worth review in order to fully understand how and when an unreachable task will be managed by Marathon.
+
+## Apache Mesos Unreachable Strategies
+
+Apache Meso's ability to communicate a task / node is unreachable is controlled by 2 concepts; 1) mesos-agent health check and 2) node rate limiter. Regarding agent health checks, the mesos-master flags of control are:
+`-max_agent_ping_timeouts` and `-agent_ping_timeout`. While the Mesos defaults are 5 and 15s respectively providing a 75 second notification event by default (assuming the loss of 1 agent). The default for DC/OS for [max_slave_ping_timeouts is 20](https://github.com/dcos/dcos/blob/9cc6ab28060545cd203c09aa7fa1b9456773d080/gen/dcos-config.yaml#L449)
+providing a 5 minute delay window between a lost node and notification to frameworks that the task is unreachable.
+
+Additionally Apache Mesos has a node rate limiter which was established prior to partition aware frameworks and is still enforced. The purpose of the rate limiter is to reduce the number of reported lost nodes / agents within a specified amount of time through the `--agent_removal_rate_limit=VALUE` flag. The default is None, which has the effect of reporting agents immediately after the health check defined above. The default in DC/OS however is [1/20mins](https://github.com/dcos/dcos/blob/9cc6ab28060545cd203c09aa7fa1b9456773d080/gen/dcos-config.yaml#L442)
```

Comment:
That's a great point, @jeschkies ; it would probably be beneficial to refer to agent is much as possible. (My understanding is node refers to the host itself, running the agent).

I believe most of the nomenclature with regards to unreachable agents is using the word agents, in the Mesos documentation.

Reply:
> My understanding is node refers to the host itself, running the agent

THat's my understanding as well.